snap/config/security-gateway-api/res/configuration.toml

```diff
@@ -0,0 +1,93 @@
+#################################################################################
```

Comment:
Just noting that the directory name (**security-gateway-api**) is a permutation of the actual git repo (**security-api-gateway**).  Same for the previous file. Was this intentional?

Reply:
Ah no, thanks for pointing this out

Reply:
All of these instances should be fixed